openquake/mbt/tests/tools/faults_test.py

```diff
@@ -25,13 +25,16 @@ def testcase01(self):
         m_upp = 7.5
         b_gr = 1.1
         bin_width = 0.05
+        # to pass test
+        m_min = 6.5
```

Comment:
I thought you were calling this parameter m_ref. I would remove the comment since it is not informative.

Reply:
No, here m_min is used correctly. In this case m_ref is m_low and this must be the modification to be added.